men_team_1/db/migrate/20181109011900_add_user_to_usertag.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserToUsertag < ActiveRecord::Migration[5.2]
+  def change
+    add_reference :usertags, :user_id, foreign_key: true
```

Comment:
Same as above: `:usertags, :user`. Also, do not put multiple migrations in the same PR. This should be combined with the first migration. You can add multiple references in the same migration (even if they are modifying different tables!), and that's exactly what you should do in this case. Delete this file.

Note: Alternatively, you could do this in two PRs if they were really logically separate migrations that you wanted to keep separate for rollback reasons, but that doesn't seem to be the case here.